README.md

```diff
@@ -34,6 +36,8 @@ buf.push_back(4);
 buf.push_back(5);
 assert_eq!(buf.to_vec(), vec![1, 2, 3, 4, 5]);
 
+// Checked addition of more elements than the buffer can contain causes error
+assert_eq!(buf.try_push_back(6).unwrap_err(), 6);
```

Comment:
Again I would avoid showcasing this in the README because this is a very special method (in my opinion) and I feel that it should appear in the documentation only. Happy to discuss this if you disagree.

Reply:
I just added "by default" to the README.